stylelint.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  root: true,
+  extends: [
+    'stylelint-config-standard',
+  ],
+};
```

Comment:
It sounds like this could be added as a "stylelint" property under package.json, if so we should do that.

Reply:
Thanks, the key stylelint works.